test/unit/css.js

```diff
@@ -860,6 +860,15 @@ QUnit.test( "Do not append px (#9548, #12990)", function( assert ) {
 	} else {
 		assert.ok( true, "No support for column-count CSS property" );
 	}
+
+	$div.css( "animation-iteration-count", 2 );
```

Comment:
@ufologist Would you like to change this line (and a few previous ones) according to what I wrote in https://github.com/jquery/jquery/pull/2793#issuecomment-168035588? I think it would be good to merge after that's done.